generators/app/templates/gruntfile_tmpl

```diff
@@ -0,0 +1,87 @@
+/**
+ * Gruntfile
+ *
+ * WARNING:
+ * Unless you know what you're doing, you shouldn't change this file.
+ * Check out the `tasks` directory instead.
+ */
+
+module.exports = function(grunt) {
+  'use strict';
+
+  /**
+   * Loads Grunt configuration modules from the specified
+   * relative path. These modules should export a function
+   * that, when run, should either load/configure or register
+   * a Grunt task.
+   *
+   * @param   {String}   dir   Directory to load
+   */
+  function loadTasks(dir) {
+    /**
+     * Resolve any relative paths
+     *
+     * @type    {String}
+     */
+    dir = require('path').resolve(dir);
+
+    /**
+     * Interal collection of modules loaded
+     *
+     * @type    {Object}
+     */
+    var modules = {};
+
+    /**
+     * Get a list of items in a directory. This is synchronous since we're
+     * only doing this once on application load
+     *
+     * @type    {Array}
+     */
+    var list = require('fs').readdirSync(dir);
+
+    /**
+     * Setup the Regex to filter out *.js files but exclude files with
+     * multiple `.` in them
+     *
+     * @type    {RegExp}
+     */
+    var jsFileTest = /^\w+\.js$/i;
```

Comment:
This regex probably needs to be updated. If the task file has a symbol in it, e.g. `-` then it wont' catch the file here.

Reply:
I modified the regex but only added the - symbol